### PR TITLE
Adds Chartbeat Analytics to Topic Page

### LINKED ISSUE #9885: Topic Tracking - Chartbeat Integration
**Is your feature request related to a problem? Please describe.**
Add Chartbeat integration to Topics.

Related to issue #https://github.com/bbc/simorgh/issues/9892

**Describe the solution you'd like**
- Add Chartbeat tracking to Topic pages (Chartbeat is already integrated on Simorgh). The page will be index in chartbeat with the following tags '{service},  {service} - Topics' For example the page https://www.bbc.com/mundo/topics/c95y3rn3gkjt will have tags 'mundo, mundo - topics'

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Checklist**
- [ ] (BBC contributors only) This issue follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Additional context**

### PR DESCRIPTION
**Pull request by pvaliani**
Resolves #9885 

**Overall change:**
Adds Chartbeat Analytics to Topic Page

**Code changes:**

- Adds Chartbeat Implementation to Topic Page
- Updates/Adds unit tests for Topic Page type
- Tidying up of some existing code for Chartbeat on other pages
---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project
- [x] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/containers/ChartbeatAnalytics/utils/index.test.js [hunk 1]
```diff
@@ -103,6 +104,11 @@ describe('Chartbeat utilities', () => {
         expectedDefaultType: PHOTO_GALLERY_PAGE,
         expectedShortType: PHOTO_GALLERY_PAGE,
       },
+      {
+        pageType: TOPIC_PAGE,
+        expectedDefaultType: 'Topics',
```
Comment (rebeccamcginn) [inline src/app/containers/ChartbeatAnalytics/utils/index.test.js:109]:
```suggestion
        expectedDefaultType: TOPIC_PAGE,
```

File: src/app/containers/ChartbeatAnalytics/utils/index.test.js [hunk 2]
```diff
@@ -103,6 +104,11 @@ describe('Chartbeat utilities', () => {
...
+        expectedShortType: 'Topics',
```
Comment (rebeccamcginn) [inline src/app/containers/ChartbeatAnalytics/utils/index.test.js:110]:
```suggestion
        expectedShortType: TOPIC_PAGE,
```

File: src/app/containers/ChartbeatAnalytics/utils/index.js [hunk 3]
```diff
@@ -96,20 +100,10 @@ export const buildSections = ({
     case MEDIA_PAGE:
       return [
         serviceCap,
-        ...(pageType
-          ? buildSectionItem(serviceCap, mediaSectionLabel[mediaPageType])
-          : []),
+        buildSectionItem(serviceCap, mediaSectionLabel[mediaPageType]),
```
Comment (jroebu14) [inline src/app/containers/ChartbeatAnalytics/utils/index.js:103]: what was the reason for removing the check for if the page type exists?

Comment (rebeccamcginn) [reply src/app/containers/ChartbeatAnalytics/utils/index.js:103]: The switch is evaluating the pageType value. If the case is MEDIA_PAGE, the page type exists. If pageType doesn't exist, we won't match this case; we'd hit `default`

**Review by ryanmccombe — APPROVED** [review]

File: src/app/containers/ChartbeatAnalytics/utils/index.js (hunk 3 above)
Comment (jroebu14) [reply src/app/containers/ChartbeatAnalytics/utils/index.js:103]: ah ok. good spot

**Review by jroebu14 — APPROVED** [review]

**Pull request merged**